main.cpp

```diff
@@ -1010,13 +730,371 @@ Part 1e - Step 19: Request a review
+*/
+
+struct Cellphone
+{
+// 5 properties: 
+    // 1) Number of camera lens (int)
+    int numCameraLens = 2;
+    //     2) Manufacturer (std::string)
+    std::string manufacaturer = "Apple";
+    //     3) Ram (int)
+    int ram = 16;
+    //     4) Screen size (float)
+    float screenSize = 6.2f;
+    //     5) Amount of storage (int)
+    int storageAmount = 32;
+    
+    // 3 things it can do:
+    //     1) Make phone calls
+    void callNumber(std::string phoneNumber);
+    //     2) Take photos and videos
+    void takePhoto(); 
+    //     3) Conenct to the internet
+    void connectOnline(std::string wifiConnection);
+};
 
+struct MovieTheater
+{
+    // 5 properties: 
+    //     1) Number of theaters (int)
+    int numTheaters = 13;
+    //     2) Cost of small popcorn (double)
+    double costSmallPopcorn = 8.9;
+    //     3) Cost of one ticket (double)
+    double costOneTicket = 19.9;
+    //     4) Number of IMAX theaters (int)
+    int numImaxTheaters = 2;
+    //     5) Number of movies available (int)
+    int numMoviesAvailable = 15;
+    // 3 things it can do:
+    //     1) Sell movie tickets
+    void chargeForTicket(); 
+    //     2) Play movies
+    void playMovie();
+    //     3) Sell food and snacks
+    float sellFoodAndSnacks (std::string itemPurchased, float fullChargeAmount); // returns items purchasing and amount charged.
+};
 
+struct Dog 
+{
+    // 5 properties:
+    //     1) Number of legs (int)
+    int numLegs = 4;
+    //     2) Name (std::string)
+    std::string name = "Bucky";
+    //     3) Age (int)
+    int age = 4;
+    //     4) Number of eyes (int)
+    int numEyes = 2;
+    //     5) Breed (std::string)
+    std::string breed = "Golden Retriever";
+    // 3 things it can do:
+    //     1) Bark
+    void bark();
+    //     2) Wag tail
+    void wagTail(std::string leftOrRight);
+    //     3) Perform a trick
+    void peformTrick (std::string trickType);
+};
 
+struct Guitar
+{
 
+    // 5 properties:
+    //     1) Number of strings (int)
+    int numStrings = 6;
+    //     2) Number of frets (int)
+    int numFrets = 22;
+    //     3) Volume level (float)
+    float volumeLevel = 3.7f;
+    //     4) Brand (std::string)
+    std::string brand = "Gibson";
+    //     5) Material (std::string)
+    std::string material = "Spruce Wood";
+
+    struct Strings
+    {
+        float lowEStringGauge = .040f; 
 
+        float dStringGauge = .010f;
+        std::string brand = "Ernie Ball";
+        std::string sku = "2229";
+        std::string material ="Nickel";
+        
+        void bend(std::string whichString);
+        void slide(std::string targetNote);
+        void snap(std::string whichString);
+    
+    };
+    // 3 things it can do:
+    //     1) Play individual notes
+    void playNote(Strings strings, std::string whichNote);
+    //     2) Be tuned to different tunings
+    void tuneString(Strings strings,std::string whichString);
```

Comment:
I'm marking this because:
1. your plain-english description is not an action, but is an ability or describes a state OR
2. the plain-english description is fine but the function name doesn't match OR
3. The way you wrote the name doesn't conform to the course coding standard:
    use `camelCase` for variable and function names.
    Type Names begin with a Capitalized first letter. All other symbols begin with a lower-case first letter.
- [ ] revise